cities_indicators/indicators/natural_areas.py

```diff
@@ -0,0 +1,37 @@
+from ..layers.esa_world_cover import EsaWorldCover
+from ..io import to_raster
+
+from xrspatial import zonal_stats
+import xarray as xr
+import numpy as np
+
+
+class NaturalAreas:
```

Comment:
I would actually define this as a subclass of EsaWorldCover where it's just mapped to these values, and then have the indicator do stats on that

Reply:
I tried to make the change. Is that what you suggested on the subclass?